component/distributed_cache/distributed_cache.go

```diff
@@ -553,9 +585,10 @@ func (dc *DistributedCache) WriteFile(options internal.WriteFileOptions) (int, e
 	log.Debug("DistributedCache::WriteFile : WriteFile, offset : %d, buf size : %d, file : %s",
 		options.Offset, len(options.Data), options.Handle.Path)
 	common.Assert(len(options.Data) != 0)
+	common.Assert(!options.Handle.IsFsDebug(), "Writing to the debug File ", options.Handle.Path)
```

Comment:
Unless the msg adds value, better to skip it. You can instead add a comment here

// Debug files are readonly.
common.Assert(!options.Handle.IsFsDebug(), options.Handle.Path)